### PR TITLE
Fixing bug where `value` URI is ignored

### PR DESCRIPTION
**Pull request by robons**
Unfortunately, due to the convoluted nature of code in the `CSVWMapping` class, interesting values in the info.json get ignored in unexpected circumstances. 

I'm fixing one of them so we listen to the `value` specified for a given column, instead of overriding it and pretending it's available locally.

### CODE REVIEW

File: gssutils/csvw/mapping.py [hunk 1]
```diff
@@ -252,7 +262,7 @@ def get_maybe_codelist_for_col(column_config: object, column_name: str) -> Optio
                     self._keys.append(self._columns[name].name)
                     self._columns[name] = self._columns[name]._replace(
                         propertyUrl=self.join_dataset_uri(f"#dimension/{pathify(name)}"),
-                        valueUrl=self.join_dataset_uri(f"#concept/{pathify(name)}/{{{self._columns[name].name}}}")
```
Comment (robons) [inline gssutils/csvw/mapping.py:255]: It used to ignore any `value` you specified here. I want it to use it please!

File: gssutils/csvw/mapping.py [hunk 2]
```diff
@@ -177,7 +177,17 @@ def get_maybe_codelist_for_col(column_config: object, column_name: str) -> Optio
                 return Resource(at_id=codelist)
 
             # Codelist should exist. Convention dictates it should be a local codelist.
-            return get_conventional_local_codelist_uri(column_name)
+            return get_conventional_local_codelist_scheme_uri(column_name)
+
+        def get_convential_local_codelist_concept_uri_template(column_name: str) -> URI:
```
Comment (rossbowen) [inline gssutils/csvw/mapping.py:182]: Typo (I think?) `convential` as opposed to `conventional`.

File: gssutils/csvw/mapping.py [hunk 3]
```diff
@@ -344,7 +354,7 @@ def get_maybe_codelist_for_col(column_config: object, column_name: str) -> Optio
                 self._keys.append(self._columns[name].name)
                 self._columns[name] = self._columns[name]._replace(
                     propertyUrl=self.join_dataset_uri(f"#dimension/{pathify(name)}"),
-                    valueUrl=self.join_dataset_uri(f"#concept/{pathify(name)}/{{{self._columns[name].name}}}")
+                    valueUrl=get_convential_local_codelist_concept_uri_template(name)
```
Comment (rossbowen) [inline gssutils/csvw/mapping.py:357]: `convential`

**Review by rossbowen — CHANGES_REQUESTED** [review]

**Review by ajtucker — APPROVED** [review]
Looks grand. Can you exhibit the bug by updating the feature file with some expectations?

**Comment by robons** [comment]
> Looks grand. Can you exhibit the bug by updating the feature file with some expectations?

I was hoping to avoid doing this, but I've committed a test now.

**Pull request merged**